### ISSUE TITLE
How to remove latency while call rest api using fetch

### ISSUE THREAD

**Issue by bdavid68** [redux-saga/redux-saga#1192]
I am using redux-saga to call a Rest API in my React Native app.

```
const response = yield call(RestApiCall, { api: url, method: 'GET', params: null });
yield put(myActionCreators.putDataRequestSuccess(response.data));

...

export async function RestApiCall({ api, method, params }) {
  ...
  return fetch('url', body);
}
```

Using this, I call several apis at once. So once each api call is finished, ui will be updated with fetched data.
But until all api calls are finished, app gets stuck and there are some latencies in user interaction.
Is there any way to remove this latency issue?

**Comment by neurosnap** [comment]
Hi, we are going to need to see how you are calling those apis at once.  If the UI has latency issues that generally means you are blocking the thread with heavy processing.  Are you doing anything that could be considered "heavy" in terms of CPU?

**Comment by Andarist** [comment]
Closing this issue due to inactivity. Feel free to respond back later here or create a new issue.

**Comment by spirobel** [comment]
this seems related: https://github.com/redux-saga/redux-saga/issues/1681#issuecomment-441359905
